### PR TITLE
Fix wildcard imports from ROOT (they are forbidden now)

### PR DESCRIPTION
**Pull request by iarspider**
#### PR description:

Following the merge of [this ROOT PR](https://github.com/root-project/root/pull/14588), replace wildcard imports from ROOT with explicit ones.

#### PR validation:

Bot tests

### CODE REVIEW

**Comment by iarspider** [comment]
label root

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
type root

**Comment by vlimant** [comment]
+1

File: DQM/SiStripMonitorClient/scripts/TH2PolyOfflineMaps.py [hunk 1]
```diff
@@ -12,9 +12,9 @@
 #Find Data files
 
 def getFileInPath(rfile):
-   import os 
-   for dir in os.environ['CMSSW_SEARCH_PATH'].split(":"):
-     if os.path.exists(os.path.join(dir,rfile)): return os.path.join(dir,rfile)                                                                          
+   import os
+   for dir in os.environ['CMSSW_SEARCH_PTH'].split(":"):
```
Comment (smuzaffar) [inline DQM/SiStripMonitorClient/scripts/TH2PolyOfflineMaps.py:16]: @iarspider , looks like a typo here `CMSSW_SEARCH_PTH`

File: DQM/SiStripMonitorClient/scripts/PixelMapPlotter.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
-#!/usr/bin/env python3 
+#!/usr/bin/env python3
 
 from __future__ import print_function
 import ROOT
 import sys
 import getopt
-from ROOT import *
+import ROOT
```
Comment (smuzaffar) [inline DQM/SiStripMonitorClient/scripts/PixelMapPlotter.py:7]: @iarspider , duplicate `import ROOT` ( see line number 4 above)

**Comment by francescobrivio** [comment]
+db

**Comment by iarspider** [comment]
please test

**Comment by smuzaffar** [comment]
@iarspider , one unit test failed https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b33758/37484/unitTests/src/DQM/SiStripMonitorClient/test/test_TreeAndTkMapProducer/testing.log
```
    from ROOT import TTree, TFile, gRoot
ImportError: cannot import name 'gRoot' from 'ROOT' (/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02824/el8_amd64_gcc12/lcg/root/6.30.05-d0417d1d3c55ded4cc9aa75628166a77/lib/ROOT/__init__.py)

```

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by perrotta** [comment]
+alca

**Comment by Martin-Grunewald** [comment]
+1

**Comment by tvami** [comment]
+1

**Comment by iarspider** [comment]
@cms-sw/dqm-l2 gentle ping.

**Comment by tjavaid** [comment]
+1

- spurious differences seen from unstable WF `12434.7` (open issue #39803 )

**Comment by rappoccio** [comment]
+1

**Pull request merged**